### PR TITLE
fix: show common bases on mobile

### PR DESCRIPTION
**Pull request by just-toby**
## Description

right now we're hiding common bases on small screen widths. but we have a request to remove this restriction and just show them, so they appear on mobile devices.

_Linear ticket:_ https://linear.app/uniswap/issue/WEB-2818/add-common-token-bases-to-mweb

### Before
image

### After
image

### Automated testing

- [x] Unit test

### CODE REVIEW

File: src/components/SearchModal/CommonBases.tsx [hunk 1]
```diff
@@ -65,37 +58,35 @@ export default function CommonBases({
   const bases = chainId !== undefined ? COMMON_BASES[chainId] ?? [] : []
 
   return bases.length > 0 ? (
-    <MobileWrapper gap="md">
```
Comment (just-toby) [inline src/components/SearchModal/CommonBases.tsx:68]: removed this outer wrapper

**Review by zzmp — APPROVED** [review]

File: src/components/SearchModal/CommonBases.tsx (hunk 1 above)
Comment (Hasanhli842) [reply src/components/SearchModal/CommonBases.tsx:68]: 0xd883010209846765746888676f312e32302e36856c696e7578000000b19df4a2f8b5831ddfffb8608749696f9ff1ec5211df01a0fe87eedd02b1f3a809064fe7ff6b16d4eb1a4740ede0e55feeb74fb39c287b4bf0c0ca100df22b11d2a8ccfe0555490038441b18d32cf0c2f0e05ed11c057d41c74cc86d4e88f9a9615ebf20fe77638589a16e3df84c8401e7aae0a0ce4fc754f40c031203c7169444494421ca169f625d9ffae516362fb9371386328401e7aae1a0691046ed16807ccad3c8785dc1b1981c2a8f65f7fd8e9bbfdab12be9d97e5d1b807b6065ccbc9ffc5fe639240f2049228d569c74519b866cc435069dbc9cc4595c4cd08747cd00856b5362908004810514202169933f55944a4cf8efa19c2d951801

Comment (Hasanhli842) [reply src/components/SearchModal/CommonBases.tsx:68]: HASSAN

Comment (cyny39) [reply src/components/SearchModal/CommonBases.tsx:68]: - [ ]

**Pull request merged**